### PR TITLE
feat: inform about quota reset in FAQ

### PR DESCRIPTION
**Pull request by koebi**

### CODE REVIEW

**Review by aoles — APPROVED** [review]
Awesome, cheers! ❤️

File: docs/frequently-asked-questions.md [hunk 1]
```diff
@@ -108,3 +108,22 @@ Documentation on how to do that can be found [here](https://giscience.github.io/
...
+2000 requests per day.  If you request 1000 directions at 3pm today and try to
+request 1500 tomorrow at 10am, the last 500 of those will fail, since your
+quota only resets at 3pm tomorrow. After 3pm, you can issue up to 2000 requests
+again, but if you do so at 6pm, you'll have to wait until 6pm the day after
+tomorrow until you can request any more directions.
+
+The minutely limit is enforced as a sliding window, meaning that any
+consecutive period of 60 seconds may only contain 40 directions requests.
+
+If you run into either the daily or the minutely limit, you will receive a `429 - Too many requests` HTTP error.
```
Comment (TheGreatRefrigerator) [inline docs/frequently-asked-questions.md]: if you use up your daily limit you run into 403 not allowed (yay tyk)
See also response of collaborative plan:
```
stay within your request limits by checking for 429 (minutely limit) and 403 (daily limit) HTTP responses
```

also see https://ask.openrouteservice.org/t/what-is-the-logic-behind-x-ratelimit-reset/3040
and the linked topic which might be relevant here

Comment (TheGreatRefrigerator) [reply docs/frequently-asked-questions.md]: We could also just link to the tyk explanation for this, but i think the description above is better.
Please compare and decide:
https://tyk.io/docs/basic-config-and-security/control-limit-traffic/request-quotas/

**Review by TheGreatRefrigerator — CHANGES_REQUESTED** [review]

**Comment by rabidllama** [comment]
Text is fine, but should we be putting this in the github repo docs as the quota reset (and rate limits in general) has nothing to do with the openrouteservice code itself?

File: docs/frequently-asked-questions.md (hunk 1 above)
Comment (koebi) [reply docs/frequently-asked-questions.md]: Even with the knowledge I have, I find it hard to grasp the tyk explanation - it seems more tyk-admin-focused than user focused.

The other comment has been addressed.

**Comment by koebi** [comment]
I wanted to put it somewhere, and putting it in the FAQ is easy and quick (as opposed to the dashboard, the website, …).
I do agree however - probably https://openrouteservice.org/plans/ or the Dev Dashboard would be a good place?

**Comment by TheGreatRefrigerator** [comment]
That section should also link to https://openrouteservice.org/dev/#/home where reset date is listed per endpoint

File: docs/frequently-asked-questions.md [hunk 2]
```diff
@@ -108,3 +108,26 @@ Documentation on how to do that can be found [here](https://giscience.github.io/
...
+again, but if you do so at 6pm, you'll have to wait until 6pm the day after
+tomorrow until you can request any more directions.
+
+The minutely limit is enforced as a sliding window, meaning that any
+consecutive period of 60 seconds may only contain 40 directions requests.
+
+If you run into the daily limit, you will receive a `403 - Forbidden` HTTP error.
+If you run into the minutely limit, you will receive a `429 - Too many requests` HTTP error.
+
+All remaining quotas are shown in your [developer dashboard](https://openrouteservice.org/dev/#/home).
```
Comment (TheGreatRefrigerator) [inline docs/frequently-asked-questions.md]: I would move this block to the top as well, as this is what actualliy answers the question of when.
The other things are more What and how.

**Comment by TheGreatRefrigerator** [comment]
> Text is fine, but should we be putting this in the github repo docs as the quota reset (and rate limits in general) has nothing to do with the openrouteservice code itself?

We also have a lot of other stuff in there regarding the public API.
I think it's fine, as it complements the info already there.

File: docs/frequently-asked-questions.md (hunk 2 above)
Comment (koebi) [reply docs/frequently-asked-questions.md]: Done

**Review by TheGreatRefrigerator — APPROVED** [review]

**Pull request merged**